start-site/src/main/resources/application.yml

```diff
@@ -1142,32 +1160,49 @@ initializr:
               href: https://cloud.spring.io/spring-cloud-static/spring-cloud-circuitbreaker/current/reference/html
         - name: Hystrix [Maintenance]
           id: cloud-hystrix
-          description: Circuit breaker with Spring Cloud Netflix Hystrix.
+          description: "Circuit breaker with Spring Cloud Netflix Hystrix. Maintenance mode, suggested replacement: Spring Cloud CircuitBreaker."
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-netflix-hystrix
           compatibilityRange: "[2.0.0.RELEASE,2.4.0-M1)"
           links:
             - rel: guide
               href: https://spring.io/guides/gs/circuit-breaker/
               description: Circuit Breaker
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-netflix/docs/2.2.6.RELEASE/reference/html/#circuit-breaker-hystrix-clients
         - name: Hystrix Dashboard [Maintenance]
           id: cloud-hystrix-dashboard
-          description: Circuit breaker dashboard with Spring Cloud Netflix Hystrix.
+          description: Circuit breaker dashboard with Spring Cloud Netflix Hystrix. In maintenance mode with no direct replacement.
           groupId: org.springframework.cloud
           artifactId: spring-cloud-starter-netflix-hystrix-dashboard
           compatibilityRange: "[2.0.0.RELEASE,2.4.0-M1)"
+          links:
+            - rel: sample
+              href: https://github.com/spring-cloud-samples/hystrix-dashboard
+              description: Sample of circuit-breaker metrics with Spring Cloud and Hystrix Dashboard
+            - rel: reference
+              href: https://docs.spring.io/spring-cloud-netflix/docs/2.2.6.RELEASE/reference/html/#circuit-breaker-hystrix-dashboard
```

Comment:
Isn't there a chance that this link will go out of sync? It's tied to a particular version.

Reply:
This module is no longer available in `3.0.0`, so we can't use `current`.